README.md

```diff
@@ -174,7 +183,7 @@ public_root_zones = {
     "vpc" = []
     "zone_id" = "Z0YYYYYYYYYYYYYYYY"
   }
-public_subdomain_default_ns_records = {
+public_delegated_secondary_ns_records = {
```

Comment:
I suppose you went with `ns_servers` as a symmetry for `ns_records`, no?

Reply:
Yes